controller/tests/models/domain/rsptag_test.py

```diff
@@ -2,6 +2,8 @@
 
 from __future__ import annotations
 
+import datetime
```

Comment:
Here too.
```suggestion
from datetime import UTC, datetime
```
and adjust the calls accordingly.